AmadeusBackend/src/main/resources/application.properties

```diff
@@ -1 +1,3 @@
 spring.application.name=AmadeusBackend
+amadeus.client.id=${MY_API_KEY}
```

Comment:
I understand this comes from an env file right? please add the steps to set it up, it it needs and env file pass etc